src/components/dashboard/Hub/index.tsx

```diff
@@ -140,13 +140,20 @@ function HubRouter() {
     }
   }
 
-  push({
-    search: showWelcome ? '?welcomescreen=true' : 'welcomescreen=false',
-  })
+  const { search } = useLocation()
+  const params = new URLSearchParams(search)
+  const skipWelcomeScreenURLParam = params.get('SkipWelcomeScreen')
+
+  useEffect(() => {
+    if (skipWelcomeScreenURLParam === 'true') {
+      setShowWelcomeScreen(false)
```

Comment:
If the skipWelcomeScreenURLParam URL query param is set to "true", then setShowWelcomeScreen to false (force skip).